libafl_sugar/Cargo.toml

```diff
@@ -17,7 +17,7 @@ all-features = true
 
 [features]
 python = ["pyo3", "libafl_qemu/python", "pyo3-build-config"]
-default = []
+default = ["libafl/adaptive_serialization"]
```

Comment:
clippy flat out refused to pass if this wasn't enabled. I'm not sure how to get around this.